examples/202-NN_examples.ipynb

```diff
@@ -55,7 +55,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    transform_date = DateFlagsTransform(day_number_in_week=True, day_number_in_month=False, out_column="dateflag")
</pre><p>Technically, there will be another feature with true value because of the default values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Reply:
It is not so important because we specify exact category columns